plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/containers/NessieContainer.java

```diff
@@ -56,6 +56,12 @@ public String getRestApiUri()
         return "http://" + getMappedHostAndPortForExposedPort(PORT) + "/api/v1";
     }
 
+    public static Map<String, String> defaultEnvVars()
+    {
+        return ImmutableMap.of("QUARKUS_HTTP_PORT", String.valueOf(PORT), "NESSIE_VERSION_STORE_TYPE",
+                VERSION_STORE_TYPE);
```

Comment:
Unwrap. The convention for method arguments is a single line or wrapping arguments per line.